### PR TITLE
feat: add datadog metric output and fix statsd config options

### LINKED ISSUE #25: Allow metrics to go to Datadog
Per #24, it'd be useful if running regularly on AWS Lambda, that the metrics could be emitted as CloudWatch data.

### PR DESCRIPTION
**Pull request by bbangert**
Add's datadog metrics output option, and fixes an issue with the
command line arguments not getting through to the statsd client.

Closes #25 

@jrconlin r?

### CODE REVIEW

File: aplt/metrics.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    """
+    def __init__(self, *args, **kwargs):
+        """Setup the metrics"""
+
+    def start(self):
+        """Start any connection needed for metric transmission"""
+
+    def stop(self):
+        """Stop any connection needed"""
+
```
Comment (jrconlin) [inline aplt/metrics.py]: No plans on using gauge?

File: aplt/runner.py [hunk 2]
```diff
@@ -313,9 +317,12 @@ def run_scenario(args=None, run=True):
 
     Usage:
         aplt_scenario WEBSOCKET_URL SCENARIO_FUNCTION [SCENARIO_ARGS ...]
-                      [--statsd_host STATSD_HOST]
-                      [--statsd_port STATSD_PORT]
-                      [--statsd_namespace STATSD_NAMESPACE]
+                      [--statsd_host=STATSD_HOST]
+                      [--statsd_port=STATSD_PORT]
+                      [--statsd_namespace=STATSD_NAMESPACE]
+                      [--datadog_api_key=DD_API_KEY]
```
Comment (jrconlin) [inline aplt/runner.py]: Do you want to note that it's either statsd or datadog?

**Comment by jrconlin** [comment]
r+

File: aplt/runner.py (hunk 2 above)
Comment (bbangert) [inline aplt/runner.py]: Ah, I do want to note that you need a namespace regardless of which one you use.

File: aplt/metrics.py (hunk 1 above)
Comment (bbangert) [inline aplt/metrics.py]: Nope, nothing in a per-client scenario would make much sense to emit a gauge of.

**Pull request merged**